front/app/containers/Admin/projects/project/timeline/edit.tsx

```diff
@@ -131,8 +131,8 @@ const AdminPhaseEdit = () => {
   useEffect(() => {
     setHasEndDate(phase?.data.attributes.end_at ? true : false);
     setAttributeDiff({});
-    setSubmitState(phase ? 'enabled' : 'disabled');
-  }, [phase]);
+    setSubmitState(phase?.data.id ? 'enabled' : 'disabled');
+  }, [phase?.data.id, phase?.data.attributes.end_at]);
```

Comment:
This was the main bug: every time the `phase` would update, the button state would be changed. Now we only do it when the phase id updates, so that during cache invalidations we don't trigger this `useEffect` block